### DISCUSSION TITLE
Xiao based accessory boards

### DISCUSSION THREAD

**Discussion by Timo614** [Seeed-Studio/OSHW-XIAO-Series#11] [Wish List]
Xiao PD Expansion board: WCH CH224K PD board with dip switch for output voltage and jumpers to connect data lines to level shifters and jumper to use the PD voltage line. Useful for controlling motors, fans, etc. that may need more voltage or current (allows up to 3A@5V). Perhaps having 2 PD controlled outputs with jumpers, 2 3v3 hy 2.0 4p i2c, serial 4p.

Xiao Solar Powered Expansion Board: Lipo rider is fine but it's too big and not the best utilization of space when using a Xiao board (can connect it vertically via a 90 degree connector for now). Ideally solder Xiao to the solar powered board (or via headers) and then have a lipo and solar connector, 2 i2c ports, 1 serial port, and another extra gpio.

Xiao Level Shifter: Board with a Xiao footprint, a bidirectional level shifters and then a few grove hy 2.0 outputs. My goal for that board would be to use it for any project where I'm using a sensor that only supports 5V or where 5V would lead to more better results (hx711 excitation voltage, etc). 

<img width="580" alt="level-up" src="https://github.com/Seeed-Studio/XIAO_Series/assets/1319304/89c9fabe-4fd4-406c-8244-bcf68b68ff2c">

Xiao ESP32S3 Sense Explorer: Utilize the https://www.hirose.com/product/p/CL0684-4098-0-51# connector to attach an alternative sensory based board in place of the camera for ESP32S3 sense. Optional flash, PDM mic, 9 dof accelerometer. Perhaps a light sensor (some of those seem pretty cheap). Haven't looked into how feasible this is.

I'll probably make a few of these myself over time just some ideas (the first three would have been useful for projects I've done in the past).

**Comment by Timo614** [comment]
I think it may be better to split it off into individual ideas if there's interest in any of them / at the moment this reads more like a wishlist of random things.

**Reply by limengdu** [reply]
great idea!
